src/common/services/UserService/UserService.ts

```diff
@@ -32,10 +32,10 @@ const readUsers = async (params: ReadUsersQueryType): Promise<ReadUsersResponseT
   }
   return await fetch(fetchQuery)
     .then((response) => {
-      if (response !== null && response.ok) {
+      try {
         return response.json();
-      } else {
-        throw new FetchError("Couldn't fetch users");
+      } catch (error) {
+        throw new FetchError("Couldn't read users");
```

Comment:
You can use `.catch` on the Promise itself. It will be triggered if promise will be rejected.